src/CoreBundle/private/library/classes/pkgShopServiceItemGiftcard/objects/TPkgShopServiceType/TPkgShopServiceType_Giftcard.class.php

```diff
@@ -26,5 +26,6 @@ public function FilterUserInput(&$aUserInput)
                 unset($aUserInput[$sKey]);
             }
         }
+        return $aUserInput;
```

Comment:
Obacht! Das ist ein harter BC Break. Code, der erwartet, dass das mitgegebene Array gefiltert wird, muss nun den Return Wert verwenden. Wenn man das nicht ändert, dann wird einfach der Input danach silently nicht mehr gefiltert. Das ist durchaus gefährlich.

Reply:
this is a signature change of the method that should explode if a user has a custom FilterUserInput method. It is documented in the migration doc- so i would risk it.

Reply:
Yes, custom implementations will explode, but callers won't. They just would lose the filtering silently. This is why I still think this is a very dangerous change.

Reply:
ok - reverted change.